org.eclipse.xtext.relengutil/src/org/eclipse/xtext/relengutil/GenerateAllExamples.mwe2

```diff
@@ -0,0 +1,36 @@
+
+module org.eclipse.xtext.relengutil.GenerateAllXbase
+
+import org.eclipse.emf.mwe.utils.*
+import org.eclipse.xtext.generator.*
+
+import org.eclipse.xtext.xtext.generator.*
+import org.eclipse.xtext.xtext.generator.model.project.*
+
+var rootPath = "../org.eclipse.xtext.xtext.ui.examples/projects"
+
+/**
+ * Must run with -Dslf4j.provider=org.slf4j.simple.SimpleServiceProvider
+ */
+Workflow {
```

Comment:
@cdietrich @LorenzoBettini Please let me know if this approach works for you

Reply:
will check in the evening

Reply:
wf works with that

Reply:
did not check diffs yet

Reply:
hmm there shouldn't be any. Curious.

Reply:
well .. I'm running with 2024-03 as TP

Reply:
same here.
there is also no diff towards your branch.

Reply:
=> if you have committed all changes we are good